cli/fmt_errors.rs

```diff
@@ -118,23 +118,31 @@ impl Deref for JSError {
 
 impl fmt::Display for JSError {
   fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    let mut formatted_frames = self.0.formatted_frames.clone();
+
+    // The formatted_frames passed from prepareStackTrace() are colored.
+    if !colors::use_color() {
+      formatted_frames = formatted_frames
+        .iter()
+        .map(|s| colors::strip_ansi_codes(s).to_string())
+        .collect();
```

Comment:
Ideally we’d do this at a lower layer... but sure I guess this fix is fine for now